jsonschema-validation.xml

```diff
@@ -1220,9 +1220,9 @@
                         their results, without asserting any conditions of their own.
```

Comment:
It seems to me that this isn't precisely true.  They _do_ assert conditions of their own, but indirectly through the result(s) of the subschemas rather than on the instance.  For example, `anyOf` asserts that at least one of its subschemas pass validation.  It's a fine line.

Reply:
Now we're getting into the semantics of what "on their own" means.

Applicators evaluate subschemas and combine / apply logic to their results.  Combining results is not asserting conditions of their own, so I'm not changing this.

Reply:
¯\\\_(ツ)_\/¯

Reply:
Maybe it's sufficient to simply add "on the instance" at the end of the sentence.

Reply:
"on the instance"?  Are you considering `true` and `false` to be non-instance assertions?  Hmm... I tend to think of them as constants, although I can't easily articulate a difference.

Really, what I want to get at is that the validation results of applicators are strictly a logical formula applied to the results of their subschemas.  The applicators themselves have no impact on the subschema's results.  In that sense, they do not "assert" specific things about the instance (or as a constant).

I suppose you could look at it as "asserting" that the result of the formula is `true`, but that's what I'm trying to avoid.  Applicators are more like operators than assertions, and that is the mindset that I want to convey.  Otherwise it's hard to see why they were split out.

Reply:
Let's just leave it, then, since you're not changing this particular thing, and people generally know what it means.